parser/yaml_output_functions.c

```diff
@@ -30,7 +30,7 @@
 #define KEY_STR_LEN 255
 #define LVL2KEY_SIZE LVL2KEY_NUM*KEY_STR_LEN
 
-#define DEBUG 0
+#define DEBUG 1
```

Comment:
Is this always going to print a bunch of stuff to the screen?

Reply:
I had this on for debugging the CI, now that it passed i'll set it back to 0.